xbmc/guilib/GUIControlGroupList.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "GUIControlGroupList.h"
 #include "input/Key.h"
-#include "GUIInfoManager.h"
+#include "guiinfo/GUIInfoLabels.h"
```

Comment:
I think the plan is to move more and more stuff out from GUIInfoManager into dedicated InfoProvider classes. So a info provider class for player related stuff, add-on related, pvr related, ...